docs/source/app_developers_guide/aws.rst

```diff
@@ -0,0 +1,427 @@
+*********************
+Using Sawtooth on AWS
+*********************
+
+.. note::
```

Comment:
The ".. note::" isn't needed now that the first sentence starts with Prerequisite.